.github/workflows/ci.yml

```
@@ -218,9 +218,29 @@ jobs:
           neon_api_key: ${{ secrets.NEON_API_KEY }}
           neon_project_id: ${{ vars.NEON_PROJECT_ID }}
           branch_limit: '10'
-          protected_branches: 'main,br-main'
+          protected_branches: 'main,development'
+          protected_creators: 'itstimwhite,Tim (user itstimwhite)'
           cleanup_threshold: '7'
 
+      - name: Guard against creating protected branch names
+        if: ${{ github.event_name == 'pull_request' && steps.check-backend.outputs.needs_db == 'true' }}
+        run: |
```

actionlint (1.7.10)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)